packages/ref-core/src/cmip_ref_core/pycmec/tests/unit/test_cmec_output.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from cmip_ref_core.pycmec.output import (
+    CMECOutput,
+)
+
+
+@pytest.fixture
+def cmec_right_output_dict():
+    return {
+        "provenance": {
+            "environment": {
+                "OS": "LINUX",
+                "Package": "ILAMB",
+                "Machine": "Frontier",
+                "Variable": "Biomass",
+            },
+            "modeldata": ['./modeldata", "./othermodels'],
+            "obsdata": {
+                "GlobalCarbon": {
+                    "version": "5.1",
+                    "title": "Global forest live biomass carbon",
+                },
+                "NBCD2000": {
+                    "version": "unknown",
+                    "name": "National Biomass and Carbon data set for the Year 2000",
+                },
+            },
+            "log": "cmec_output.log",
```

Comment:
@nocollier As mentioned on slack the CMEC output bundle has a `log` field which takes a log file